scripts/dev_setup.sh

```diff
@@ -286,6 +303,44 @@ function install_openssl_dev {
   fi
 }
 
+function install_lcov {
+  PACKAGE_MANAGER=$1
+  #Differently named packages for lcov with different sources.
+  if [[ "$PACKAGE_MANAGER" == "apk" ]]; then
+    apk --update add --no-cache  -X http://dl-cdn.alpinelinux.org/alpine/edge/testing lcov
+  fi
+  if [[ "$PACKAGE_MANAGER" == "apt-get" ]]; then
+    install_pkg lcov "$PACKAGE_MANAGER"
+  fi
+  if [[ "$PACKAGE_MANAGER" == "yum" ]] || [[ "$PACKAGE_MANAGER" == "dnf" ]]; then
+    install_pkg lcov "$PACKAGE_MANAGER"
+  fi
+  if [[ "$PACKAGE_MANAGER" == "pacman" ]] || [[ "$PACKAGE_MANAGER" == "brew" ]]; then
+    echo nope no lcov for you.
+    # install_pkg git "$PACKAGE_MANAGER"
```

Comment:
Probably more useful to echo this block than as comment here.

Reply:
hmm good idea.